tmou-mapa-backend/src/main.rs

```diff
@@ -447,6 +447,112 @@ fn admin_standings(_admin: Admin, conn: PostgresDbConn) -> Result<Json<api::Stan
     }
 }
 
+////////////////
+// item editing
+////////////////
+
+#[allow(unused)]
+#[get("/admin/items")]
```

Comment:
Přemýšlím, jestli není dobrý čas udělat nějakou samostatné `routes` soubory. `main.rs` začíná být nějaký plný. Ale vůbec nevím, jak se to dělá.